TACC/Core-Portal
# task/FP-1591: Add pytorch icon for pytorch resnet frontera app

## Overview

Updated cortal icons to v1.5 to include pytorch icon for KFAC-PyTorch Frontera app

## Related

* [FP-1591](https://jira.tacc.utexas.edu/browse/FP-1591)

## Changes

- Updated trumps/icon.fonts.css to cortal icons v1.5


## Testing

1. Verify no icons are missing on portal
2. Add KFAC-PyTorch app:
  - Go to https://cep.dev/core/admin/workspace/apptrayentry/ and go to 'Add App Tray Entry'
  - Label app 'KFAC-PyTorch'
  - Add app to category 'Data Processing'
  - For icon, put 'pytorch'
  - For appId, put 'pytorch-resnet-frontera-0.3.1u1'
3. Navigate back to Dashboard -> Applications -> Data Processing, and verify KFAC-PyTorch app has PyTorch icon in black (see notes for image in color)  

## UI
<img width="1204" alt="image" src="https://user-images.githubusercontent.com/43251554/186013619-6389c712-7e33-44e3-b083-1676aa9bf450.png">



## Notes

![image](https://user-images.githubusercontent.com/43251554/186013346-4a7a3ee7-1534-4628-bdae-1b2d22cbd9fa.png)
